integration-libs/epd-visualization/src/components/visual-picking/visual-picking-product-filter/visual-picking-product-filter.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="form-group search-wrapper">
+  <input
+    #queryInput
```

Comment:
it seems this id is not used later in this file. Do we really need it?

Reply:
It is not needed. I will remove it.